pkg/catalog/manager/traverse.go

```diff
@@ -20,7 +20,36 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 )
 
+func (m *Manager) dropDeprecatedFields(cmt *CatalogInfo, catalogType string) error {
```

Comment:
Does it make sense to move this new helper func to somewhere after `traverseAndUpdate`? Another helper func called by `traverseAndUpdate` (`processInvalidChartErrors`) is all the way at the bottom of the file.

Reply:
I will move it